module/advancement/scale-value.js

```diff
@@ -257,6 +257,17 @@ export class ScaleValueConfig extends AdvancementConfig {
   activateListeners(html) {
     super.activateListeners(html);
     this.form.querySelector("input[name='data.title']").addEventListener("input", this._onChangeTitle.bind(this));
+    this.form.querySelector("i[name='identifier-hint-copy']").addEventListener("click", this._onIdentifyerHintCopy.bind(this));
+  }
+
+  /* -------------------------------------------- */
+
+  /**
+   * Copies the full scale identifier hint to the clipboard.
+   */
+  _onIdentifyerHintCopy() {
+    let data = this.getData();
+    navigator.clipboard.writeText(`@scale.${data.classIdentifier}.${data.previewIdentifier}`);
```

Comment:
Showing a notification when the copy occurs here would be helpful:

```suggestion
    const identifier = `@scale.${data.classIdentifier}.${data.previewIdentifier}`;
    navigator.clipboard.writeText(identifier);
    ui.notifications.info(game.i18n.format("DND5E.IdentifierCopied", { identifier }));
```

```
"DND5E.IdentifierCopied": "Identifier {identifier} copied to clipboard",
```

Reply:
Hm, rather than an entire notification for what is a relatively small operation, might something similar to what GitHub does be more appropriate?
![image](https://user-images.githubusercontent.com/524596/185470465-14202981-d3ab-4923-9d77-6148804b81e5.png)

I think just changing the icon temporarily to a check-mark would be sufficient. Plus, it requires less translation.

Reply:
That would work.

You could to try and use the new [`TooltipManager`](https://foundryvtt.com/api/v10/classes/client.TooltipManager.html) in V10 to easily show a little balloon.

Reply:
Core foundry uses a `ui.notifications.info` for its clipboard copied message for document IDs so I think it's ok (and more consistent) to do the same here.